packages/react-core/src/components/Select/Select.tsx

```diff
@@ -44,6 +44,8 @@ export interface SelectProps
   noResultsFoundText?: string;
   /** Selected item for single select variant.  Array of selected items for multi select variants. */
   selections?: string | SelectOptionObject | (string | SelectOptionObject)[];
+  /** Flag indicating if selection badge should be hidden for checkbox variant,default false */
+  hideCheckboxSelectionBadge?: boolean;
```

Comment:
Or flag variables usually start with "is" or "are". So maybe update this to `isCheckboxSelectionBadgeHidden`